pkg/csi/controller_server.go

```diff
@@ -55,8 +59,28 @@ func NewControllerServer(coreClient ctlv1.Interface, virtClient kubecli.Kubevirt
 	}
 }
 
+func (cs *ControllerServer) isStorageClassValid(storageClassName string) (bool, error) {
+	logrus.Infof("Prepare to check the host StorageClass: %s", storageClassName)
+	storageClassInst, err := cs.storageClient.StorageClass().Get(storageClassName, metav1.GetOptions{})
+	if err != nil {
+		errReason := errors.ReasonForError(err)
+		switch errReason {
+		case metav1.StatusReasonForbidden:
+			logrus.Warnf("No permission, skip checking. err: %+v", err)
```

Comment:
It would be nice to add some comments on why it's OK to continue in this case.

Reply:
done